templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/{{cookiecutter.lowercase_modelname}}.rst

```diff
@@ -10,7 +10,7 @@
     an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
     specific language governing permissions and limitations under the License.
 
-{{cookiecutter.uppercase_modelname}}
+{{cookiecutter.camelcase_modelname}}
```

Comment:
As @sgugger fixed in https://github.com/huggingface/transformers/pull/9910, it should actually be `modelname`!